### PR TITLE
RavenDB-14978 Add load balance behavior

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/wwwroot/Content/css/pages/client-configuration.less [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#client-configuration-view {
+
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/pages/client-configuration.less]: why we need that?

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/Content/css/pages/client-configuration.less]: just because I used it in the _less_ file

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/pages/client-configuration.less]: ?

File: src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts [hunk 2]
```diff
@@ -8,34 +9,62 @@ class clientConfigurationModel {
...
-    disabled = ko.observable<boolean>();
-
-    readBalanceBehaviorLabel = ko.pureComputed(() => {
-        const value = this.readBalanceBehavior();
-        const kv = clientConfigurationModel.readModes;
-        return value ? kv.find(x => x.value === value).label : "None";
-    });
+    useSessionContextForLoadBehavior = ko.observable<Raven.Client.Http.LoadBalanceBehavior>();
     
+    readBalanceBehavior = ko.observable<Raven.Client.Http.ReadBalanceBehavior>('None');
```
Comment (ml054) [inline src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts]: please use " whenever possible

File: src/Raven.Studio/typescript/viewmodels/database/settings/clientConfiguration.ts [hunk 3]
```diff
@@ -90,6 +57,51 @@ class clientConfiguration extends viewModelBase {
...
+
+        this.effectiveMaxNumberOfRequestsPerSession = ko.pureComputed(() => {
+            if (!this.hasGlobalConfiguration()) {
+                return "";
+            }
+
+            const configToUse = this.overrideServer() ? this.model : this.globalModel;
+            const maxRequests = configToUse.maxNumberOfRequestsPerSession();
+
+            return _.includes(configToUse.isDefined(), "maxNumberOfRequestsPerSession") && maxRequests ? maxRequests.toLocaleString() :  '<Client Default>';
```
Comment (ml054) [inline src/Raven.Studio/typescript/viewmodels/database/settings/clientConfiguration.ts]: use "

Comment (Danielle9897) [reply src/Raven.Studio/typescript/viewmodels/database/settings/clientConfiguration.ts]: done

File: src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/models/database/settings/clientConfigurationModel.ts]: done

**Review by ml054 — APPROVED** [review]

**Pull request merged**